addon-test-support/audit.js

```diff
@@ -66,10 +66,41 @@ Ember.Test.registerAsyncHelper('a11yAudit', function(app, ...args) {
  * @method a11yAudit
  * @public
  */
-export default function a11yAudit(...args) {
+function a11yAudit(...args) {
   if (window.a11yAudit) {
     return window.a11yAudit(...args);
   }
 
   return runA11yAudit(...args);
 }
+
+/**
+ * A wrapper method to run the a11yAudit if desired
+ *
+ * @method a11yAuditIf
+ * @public
+ */
+function a11yAuditIf(...args) {
+  if(getUrlParameter('enableA11yAudit') === 'true') {
+    return a11yAudit(...args);
+  }
+  return RSVP.resolve('no operation');
+}
+
+/**
+ * A method to return the value of queryParameter
+ *
+ * @method getUrlParameter
+ * @private
+ */
+function getUrlParameter(name) {
+    name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
+    var regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
+    var results = regex.exec(location.search);
+    return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
+};
+
+export {
```

Comment:
Changing these exports will be a breaking change, meaning we would need to release a new major version, which I would rather not do. Instead, we should extract the new function into it's own module.